play-json/shared/src/test/scala/play/api/libs/json/MacroSpec.scala

```diff
@@ -655,7 +661,6 @@ class MacroSpec extends WordSpec with MustMatchers
     def id(value: String): Id = value.asInstanceOf[Id]
 
     implicit val idReads: Reads[Id] = implicitly[Reads[String]].asInstanceOf[Reads[Id]]
-    implicit val idWrites: Writes[Id] = implicitly[Writes[String]]
```

Comment:
No longer required, implicitly provided by the new [stringableWrites](https://github.com/playframework/play-json/pull/216/files#diff-43f2e42c6ce469bcd932a93a187413feR418)